spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -177,8 +177,13 @@ def place(self, place_target: str = None) -> Tuple[bool, str]:
         else:
             return False, "Place failed to reach the target position"
 
+    def get_env(self):
+        "Get the env for the ease of the access"
+        return self.nav_controller.nav_env
+
     def dock(self):
-        # TODO: Stow back the arm
+        # Stow back the arm
+        self.get_env().reset_arm()
```

Comment:
Thanks for fixing this TODO, looks good. In future, let's add a few default configurations for arm -- READY, STOW, CARRY, etc and add a method in spot.py to bring arm to one of the defaults as per the need.